worlds/fe8/data/characters.json

```diff
@@ -1,65 +1,65 @@
-    "Lute": {"ids": [12], "tags": ["no_generic"]},
-    "Natasha": {"ids": [13], "tags": ["no_generic"]},
-    "Cormag": {"ids": [14], "tags": ["no_generic"]},
-    "Ephraim": {"ids": [15], "tags": ["no_generic"]},
-    "Forde": {"ids": [16], "tags": ["no_generic"]},
-    "Kyle": {"ids": [17], "tags": ["no_generic"]},
-    "Amelia": {"ids": [18], "tags": ["no_generic"]},
-    "Artur": {"ids": [19], "tags": ["no_generic"]},
-    "Gerik": {"ids": [20], "tags": ["no_generic"]},
-    "Tethys": {"ids": [21], "tags": ["no_generic"]},
-    "Marisa": {"ids": [22], "tags": ["no_generic"]},
-    "Saleh": {"ids": [23], "tags": ["no_generic","must_fight"]},
-    "Ewan": {"ids": [24], "tags": ["no_generic"]},
-    "L'Arachel": {"ids": [25], "tags": ["no_generic"]},
-    "Dozla": {"ids": [26], "tags": ["no_generic"]},
-    "Rennac": {"ids": [28], "tags": ["no_generic"]},
-    "Duessel": {"ids": [29], "tags": ["no_generic"]},
-    "Myrrh": {"ids": [30], "tags": ["no_generic"]},
-    "Knoll": {"ids": [31], "tags": ["no_generic","must_fight"]},
-    "Joshua": {"ids": [32], "tags": ["no_generic"]},
-    "Syrene": {"ids": [33], "tags": ["no_generic"]},
-    "Tana": {"ids": [34], "tags": ["no_generic"]},
-    "Lyon": {"ids": [35, 64, 108], "tags": ["no_generic"]},
-    "Orson": {"ids": [36, 66, 109], "tags": ["no_generic"]},
-    "Glen": {"ids": [37, 105], "tags": ["no_generic"]},
-    "Selena": {"ids": [38, 68], "tags": ["no_generic"]},
-    "Valter": {"ids": [39, 67, 69], "tags": ["no_generic"]},
-    "Riev": {"ids": [40, 87], "tags": ["no_generic"]},
-    "Caellach": {"ids": [41, 83], "tags": ["no_generic"]},
-    "Pablo": {"ids": [79, 84], "tags": ["no_generic"]},
-    "Morva": {"ids": [65], "tags": ["no_generic"]},
-    "Breguet": {"ids": [70], "tags": []},
-    "Bone": {"ids": [71], "tags": []},
-    "Bazba": {"ids": [72], "tags": []},
-    "C4Entombed": {"ids": [73], "tags": []},
-    "Saar": {"ids": [74], "tags": []},
-    "Novala": {"ids": [75], "tags": []},
-    "Murray": {"ids": [76], "tags": []},
+    "Eirika": {"ids": [1], "tags": []},
+    "Seth": {"ids": [2], "tags": []},
+    "Gilliam": {"ids": [3], "tags": []},
+    "Franz": {"ids": [4], "tags": []},
+    "Moulder": {"ids": [5], "tags": []},
+    "Vanessa": {"ids": [6], "tags": []},
+    "Ross": {"ids": [7], "tags": []},
+    "Neimi": {"ids": [8], "tags": []},
+    "Colm": {"ids": [9], "tags": []},
+    "Garcia": {"ids": [10], "tags": []},
+    "Innes": {"ids": [11], "tags": []},
+    "Lute": {"ids": [12], "tags": []},
+    "Natasha": {"ids": [13], "tags": []},
+    "Cormag": {"ids": [14], "tags": []},
+    "Ephraim": {"ids": [15], "tags": []},
+    "Forde": {"ids": [16], "tags": []},
+    "Kyle": {"ids": [17], "tags": []},
+    "Amelia": {"ids": [18], "tags": []},
+    "Artur": {"ids": [19], "tags": []},
+    "Gerik": {"ids": [20], "tags": []},
+    "Tethys": {"ids": [21], "tags": []},
+    "Marisa": {"ids": [22], "tags": []},
+    "Saleh": {"ids": [23], "tags": []},
```

Comment:
Is there a reason we dropped `must_fight` here?

Reply:
I don't think that one really does anything? He should already set to `must_fight` in every significant unit block.